Ginger/Ginger/GeneralWindows/HelpLayout/HelpLayoutArgs.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Windows;
+
+namespace Ginger.GeneralWindows
+{
+    public class HelpLayoutArgs
+    {
+        //public enum eEventType
+        //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4555652)